mne/morph.py

```diff
@@ -416,6 +416,8 @@ def save(self, fname, overwrite=False, verbose=None):
 def _check_subject_from(subject_from, src):
     if isinstance(src, string_types):
         subject_check = src
+    elif src is None:  # assume it's correct although dangerous but unlikely
+        subject_check = subject_from
```

Comment:
if you have an stc you give as src in compute_source_morph and this stc has no subject then you now assume it's coming from subject_from and don't break